client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.js

```diff
@@ -113,6 +128,12 @@ const DataFilesToolbar = ({ scheme, api }) => {
   );
   const isPrivate = ['projects', 'private'].includes(scheme);
   const canRename =
+    selectedFiles.length === 1 &&
+    isPrivate &&
+    api !== 'googledrive' &&
```

Comment:
Not relevant to this PR, but I was thinking out loud. If we want more "plug and play" functionality of the storage systems and external resources like dropbox/google drive, is it practical to check against values pulled from settings?

Reply:
I think that's a good point actually, we can instead look at `tapis` being the api